src/components/WorkPoolDetails.vue

```diff
@@ -52,7 +54,10 @@
   })
   const schema = computed(() => props.workPool.baseJobTemplate.variables ?? {})
   const showBaseJobTemplateDetails = computed(() => props.workPool.type && schemaHasProperties.value && can.access.workers)
-  const baseJobTemplateVariablesDefaults = getSchemaDefaults(props.workPool.baseJobTemplate.variables ?? {})
+  const baseJobTemplateVariablesDefaults = computed(() => {
+    const defaults = getSchemaDefaults(props.workPool.baseJobTemplate.variables ?? {})
+    return schemaValuesBlockReferencesWithSchemaResolver(defaults, cloneDeep(schema.value))
```

Comment:
I don't want to have to use this `cloneDeep`, but I get an infinite update loop if I don't use it, so `schema` must be getting updated in this function call. Is there a better way to do this?